src/compiler/factory/nodeFactory.ts

```diff
@@ -3776,6 +3778,17 @@ namespace ts {
                 : node;
         }
 
+        // @api
+        function createImportCall(importKeyword: Token<SyntaxKind.ImportKeyword> | undefined, typeArguments: readonly TypeNode[] | undefined, argumentsArray: readonly Expression[]) {

```

Comment:
Since `ImportCall` doesn't have its own syntax kind, this belongs down below where we have functions like `createImmediatelyInvokedFunctionExpression`. I'm trying to keep the sources in this file aligned with the ordering of `SyntaxKind` to make it easier to navigate the file.

Also, I'd recommend you replace `Token<SyntaxKind.ImportKeyword>` with `ImportExpression`:
```suggestion
        function createImportCall(importKeyword: ImportExpression | undefined, typeArguments: readonly TypeNode[] | undefined, argumentsArray: readonly Expression[]) {
```

We could probably add a `createImportExpression` that returns `ImportExpression` similar to how we have `createSuper` and `createThis` (which I should have renamed to `createSuperExpression` and `createThisExpression` when the factory API was added, an oversight on my part).